yarn.lock

```diff
@@ -1181,21 +1181,6 @@
     minimatch "^3.0.4"
     strip-json-comments "^3.1.1"
 
-"@fluentui-react-native/framework@0.5.4":
```

Comment:
This is a lockfile change that is a result of a different change, I think. Pretty sure it's already in the repo

Reply:
I've seen this hop back and forth in different reviews. It might be worth double checking it does exist. If it doesn't I'm sure the next person who runs `yarn` will add it back.

Reply:
Wait.. should this even be here given that this is a local package?

Reply:
uh... great question? Anyway I just merged master and it didn't come back, so I assume this is just gone